engines/director/detection_tables.h

```diff
@@ -74,6 +74,7 @@ static const PlainGameDescriptor directorGames[] = {
 	{ "barbpauper",			"Barbie as the Princess and the Pauper" },
 	{ "barbrapunzel",		"Barbie as Rapunzel: A Creative Adventure" },
 	{ "barbswanlake",		"Barbie of Swan Lake: The Enchanted Forest" },
+	{ "barbmagichouse", "Barbie and her Magical House"},
```

Comment:
Is this aligned vertically with the rest of the lines?

Reply:
Also this should be above "barbnail" to maintain alphabetical order